src/views/LimitedRestorationToFull.vue

```diff
@@ -180,11 +192,19 @@ export default class LimitedRestorationToFull extends Vue {
   @Action setResource!: ActionBindingIF
   @Action setRestorationRelationships!: ActionBindingIF
   @Action setStateFilingRestoration!: ActionBindingIF
+  @Action setValidComponent!: ActionBindingIF
+
+  @Mutation mutateRestorationCourtOrder!: ActionBindingIF
 
   /** Whether App is ready. */
   @Prop({ default: false }) readonly appReady!: boolean
   @Prop({ default: 0 }) readonly restorationId!: number
 
+  /* isDataLoaded is a flag that is to "true" after the component's data loaded
+  it's used a work-around because the shared components aren't reactive to
+  changes after the shared components are mounted. */
```

Comment:
If you use JSDoc syntax here then hovering over the property gives you its comment :)

Reply:
Is there a ticket to fix this later?

Reply:
Good point, I'll change to use the JSDoc syntax.

Reply:
Okay to resolve?

Reply:
Weird, the comment has been changed to a JSDoc comment, but Github does not show the change.

Reply:
Probably the file didn't get committed.

Reply:
> Probably the file didn't get committed.

On the PR's "conversation" tab the comment shows as not changed, but on the "files" tab, the changes are shown.  I promise I didn't forget to commit the file!  🤞

Reply:
🤷‍♂️